README.md

```diff
@@ -28,6 +28,9 @@ http://entrproject.org/) utility:
 ```bash
 find . -name "*_shpec.sh" | entr shpec
 ```
+__Warning__: If you use `zsh` you should disable the `end` keyword. (to do that, run `disable -r end`.
```

Comment:
Good Idea. Then the readMee would just have a 'For blablabla compatibility, just refer the ....'

Then the questions are:

- what name for the document
- what structure? 
  (intro, Bash, Zsh ,..., (then specific substructure for each shell)